testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -222,11 +223,11 @@ class MiniAppDisplayActivity : BaseActivity() {
 
             override fun getPoints(
                 onSuccess: (points: Points) -> Unit,
-                onError: (message: String) -> Unit
+                onError: (pointsError: MiniAppPointsError) -> Unit
             ) {
                 val points = AppSettings.instance.points
                 if (points != null) onSuccess(points)
-                else onError("There is no points found in HostApp.")
+                else onError(MiniAppPointsError(null, "There is no points found in HostApp."))
```

Comment:
We can use `MiniAppPointsError.custom(""There is no points found in HostApp.")` like this? or make the type nullable?

Reply:
Updated.